content/influxdb/cloud-serverless/get-started/write.md

```diff
@@ -322,226 +329,379 @@ dependencies to your current project.
-    to InfluxDB. _Save the script to a file and run `python SCRIPT_NAME` or run `python` to write and execute the script using the interactive shell._
+4.  Create a file for your code--for example, `write.py`.
+
+    ```sh
+    touch write.py
+    ```
 
-    1.  Import the `InfluxDBClient3` object from the `influxdb_client_3` module.
-    2.  Use the `InfluxDBClient3` constructor to instantiate an InfluxDB Client.
-        The example below assigns it to the `client` variable.
-        Provide the following credentials:
+5.  Inside of `write.py`, enter the following sample code:
+
+      ```py
+      from influxdb_client_3 import InfluxDBClient3
+
+      # INFLUX_TOKEN is an environment variable you assigned to your
+      # API token value.
+      token = os.getenv('INFLUX_TOKEN')
+
+      # host is the URL without protocol or trailing slash
+      client = InfluxDBClient3(
+          host='cloud2.influxdata.com',
+          org='',
+          token=token,
+          database='get-started'
+      )
+
+      lines = [
+          "home,room=Living\ Room temp=21.1,hum=35.9,co=0i 1641024000",
+          "home,room=Kitchen temp=21.0,hum=35.9,co=0i 1641024000",
+          "home,room=Living\ Room temp=21.4,hum=35.9,co=0i 1641027600",
+          "home,room=Kitchen temp=23.0,hum=36.2,co=0i 1641027600",
+          "home,room=Living\ Room temp=21.8,hum=36.0,co=0i 1641031200",
+          "home,room=Kitchen temp=22.7,hum=36.1,co=0i 1641031200",
+          "home,room=Living\ Room temp=22.2,hum=36.0,co=0i 1641034800",
+          "home,room=Kitchen temp=22.4,hum=36.0,co=0i 1641034800",
+          "home,room=Living\ Room temp=22.2,hum=35.9,co=0i 1641038400",
+          "home,room=Kitchen temp=22.5,hum=36.0,co=0i 1641038400",
+          "home,room=Living\ Room temp=22.4,hum=36.0,co=0i 1641042000",
+          "home,room=Kitchen temp=22.8,hum=36.5,co=1i 1641042000",
+          "home,room=Living\ Room temp=22.3,hum=36.1,co=0i 1641045600",
+          "home,room=Kitchen temp=22.8,hum=36.3,co=1i 1641045600",
+          "home,room=Living\ Room temp=22.3,hum=36.1,co=1i 1641049200",
+          "home,room=Kitchen temp=22.7,hum=36.2,co=3i 1641049200",
+          "home,room=Living\ Room temp=22.4,hum=36.0,co=4i 1641052800",
+          "home,room=Kitchen temp=22.4,hum=36.0,co=7i 1641052800",
+          "home,room=Living\ Room temp=22.6,hum=35.9,co=5i 1641056400",
+          "home,room=Kitchen temp=22.7,hum=36.0,co=9i 1641056400",
+          "home,room=Living\ Room temp=22.8,hum=36.2,co=9i 1641060000",
+          "home,room=Kitchen temp=23.3,hum=36.9,co=18i 1641060000",
+          "home,room=Living\ Room temp=22.5,hum=36.3,co=14i 1641063600",
+          "home,room=Kitchen temp=23.1,hum=36.6,co=22i 1641063600",
+          "home,room=Living\ Room temp=22.2,hum=36.4,co=17i 1641067200",
+          "home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200"
+      ]
+
+      client.write(lines,write_precision='s')
+      ```
```

Comment:
This needs to be wrapped in the `influxdb/custom-timestamps` shortcode.

~~~suggestion
      {{% influxdb/custom-timestamps %}}
      ```py
from influxdb_client_3 import InfluxDBClient3

# INFLUX_TOKEN is an environment variable you assigned to your
# API token value.
token = os.getenv('INFLUX_TOKEN')

# host is the URL without protocol or trailing slash
client = InfluxDBClient3(
    host='cloud2.influxdata.com',
    org='',
    token=token,
    database='get-started'
)

lines = [
    "home,room=Living\ Room temp=21.1,hum=35.9,co=0i 1641024000",
    "home,room=Kitchen temp=21.0,hum=35.9,co=0i 1641024000",
    "home,room=Living\ Room temp=21.4,hum=35.9,co=0i 1641027600",
    "home,room=Kitchen temp=23.0,hum=36.2,co=0i 1641027600",
    "home,room=Living\ Room temp=21.8,hum=36.0,co=0i 1641031200",
    "home,room=Kitchen temp=22.7,hum=36.1,co=0i 1641031200",
    "home,room=Living\ Room temp=22.2,hum=36.0,co=0i 1641034800",
    "home,room=Kitchen temp=22.4,hum=36.0,co=0i 1641034800",
    "home,room=Living\ Room temp=22.2,hum=35.9,co=0i 1641038400",
    "home,room=Kitchen temp=22.5,hum=36.0,co=0i 1641038400",
    "home,room=Living\ Room temp=22.4,hum=36.0,co=0i 1641042000",
    "home,room=Kitchen temp=22.8,hum=36.5,co=1i 1641042000",
    "home,room=Living\ Room temp=22.3,hum=36.1,co=0i 1641045600",
    "home,room=Kitchen temp=22.8,hum=36.3,co=1i 1641045600",
    "home,room=Living\ Room temp=22.3,hum=36.1,co=1i 1641049200",
    "home,room=Kitchen temp=22.7,hum=36.2,co=3i 1641049200",
    "home,room=Living\ Room temp=22.4,hum=36.0,co=4i 1641052800",
    "home,room=Kitchen temp=22.4,hum=36.0,co=7i 1641052800",
    "home,room=Living\ Room temp=22.6,hum=35.9,co=5i 1641056400",
    "home,room=Kitchen temp=22.7,hum=36.0,co=9i 1641056400",
    "home,room=Living\ Room temp=22.8,hum=36.2,co=9i 1641060000",
    "home,room=Kitchen temp=23.3,hum=36.9,co=18i 1641060000",
    "home,room=Living\ Room temp=22.5,hum=36.3,co=14i 1641063600",
    "home,room=Kitchen temp=23.1,hum=36.6,co=22i 1641063600",
    "home,room=Living\ Room temp=22.2,hum=36.4,co=17i 1641067200",
    "home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200"
]

client.write(lines,write_precision='s')
```
      {{% /influxdb/custom-timestamps %}}
~~~